project/phoenix.py

```diff
@@ -0,0 +1,19 @@
+from . import BaseProject, TEMPLATEDIR
+import os
+
+
+class Phoenix_elixir(BaseProject):
+    '''Phoenix_elixir object.
+
+    This is necessary because elixir has some very specific naming conventions, that prevent the template from being called
```

Comment:
FFS, Elixir!  Have you learned nothing from Go?

Reply:
no way jose